src/components/page/Header/index.tsx

```diff
@@ -216,6 +219,19 @@ export class HeaderComponent extends React.Component<HeaderProps, State> {
                         </DropdownItem>
                       </div>
                     )}
+                    {ENABLE_DYNAMIC_MDA && (
```

Comment:
should we also include this in the check of when the `monitor`-tab dropdown is rendered in the header navigation?